Core.Collectors/IO/AzureHelpers.cs

```diff
@@ -118,6 +125,27 @@ public static async Task<CloudQueue> GetStorageQueueCachedAsync(string queueName
             }
         }
 
+        public static async Task<CloudQueue> GetStorageQueueCachedUsingMsiAsync(string queueName, string storageAccountNameEnvironmentVariable = "StorageAccountName")
```

Comment:
>storageAccountNameEnvironmentVariable = "StorageAccountName" [](http://example.com/codeflow?start=105&length=60)

Does using a default here even make any sense? In other words, if this does not get overridden, would it do anything sensical or would that crash? If the latter, let's start enforcing passing this parameter without a default.